marklogic-data-hub-central/ui/src/components/facet/facet.tsx

```diff
@@ -169,7 +169,7 @@ const Facet: React.FC<Props> = (props) => {
           >Clear
           </div>
           <div className={styles.toggle} onClick={() => toggleShow(!show)} data-testid={stringConverter(props.name) + "-toggle"}>
-            <Icon style={{ fontSize: '12px' }} type='down' rotate={show ? 0 : 180} />
+            <Icon style={{ fontSize: '14px' }} type='down' rotate={show ? 0 : -90} />
```

Comment:
same change as mentioned in another comment. Let's move all the CSS styles to the corresponding module.scss file.